packages/rich-text/src/plugins/CommandPalette/components/CommandList.tsx

```diff
@@ -96,6 +96,15 @@ export const CommandList = ({ query, editor }: CommandListProps) => {
   const commandItems = useCommands(sdk, query, editor);
   const { selectedItem } = useCommandList(commandItems, container);
 
+  React.useEffect(() => {
+    editor.tracking.onCommandPaletteAction('openRichTextCommandPalette');
+
+    return () => {
+      editor.tracking.onCommandPaletteAction('closeRichTextCommandPalette');
```

Comment:
This is `cancelRichTextCommandsPalette` not close. I think it should be called when pressing `esc` instead?

Reply:
Shouldn't `close` be called on any dismount, since that will cover both the palette closing due to the user not wanting it (cancel) and also after embedding? Or do we want to add a `cancel` event?